### PR TITLE
Zach/gallery

### LINKED ISSUE #12: Create Gallery Header
Screenshot 2024-10-06 at 1 47 24 AM

edit in : src/components/gallery/Gallery.tsx

### PR DESCRIPTION
**Pull request by ZacharyTa**
Added GalleryHeader component in _'src/components/gallery/GalleryHeader.tsx'_
    - Copied Gallery Header's font properties from Figma using tailwind

Added Gallery page in _'src/app/gallery/page.tsx'_

### CODE REVIEW

**Comment by JaviChip** [comment]
Hey can you add a screen shot of what it looks like ? Thanks !

**Comment by ZacharyTa** [comment]
Here is what it looks like: image
I didn't make the text white because Im assuming the background color will be black later.
Let me know if you have any further questions! Thank you

File: src/components/gallery/GalleryHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const GalleryHeader = () => {
+  return (
+    <div className="font-banco flex w-full flex-row justify-center text-left text-[108px] font-medium leading-[103.25px]">
```
Comment (JaviChip) [inline src/components/gallery/GalleryHeader.tsx]: Can you change text-white , the main page is updated so it will be fine

Comment (JaviChip) [inline src/components/gallery/GalleryHeader.tsx]: Instead of text-108px , can you use established sizes 

i.e. => text-3xl , text-2xl etc

Comment (JaviChip) [inline src/components/gallery/GalleryHeader.tsx]: What does leading- do ?

 I dont think its needed

Comment (ZacharyTa) [reply src/components/gallery/GalleryHeader.tsx]: Ah it sets the line height of the text, I was trying to copy the line height typography properties from the Figma design, but yea you're right its definitely not needed in this. I'll remove it

Comment (ZacharyTa) [reply src/components/gallery/GalleryHeader.tsx]: Alrighty! I changed it to `text-8xl` bc it seems to be the closest font text size as text-108px

**Comment by ZacharyTa** [comment]
Screenshot: Updated Gallery Header
image

**Review by JaviChip — APPROVED** [review]

**Pull request merged**